ludwig/utils/backward_compatibility.py

```diff
@@ -90,8 +91,9 @@ def upgrade_to_latest_version(config: Dict):
             "ludwig_version".
     """
     if "ludwig_version" in config:
+        to_version = re.findall(r".*\d", LUDWIG_VERSION)[0]
```

Comment:
can `re.findall` return an empty list? if so, we need to check before indexing with `[0]`